lib/commands/network.js

```diff
@@ -38,7 +38,8 @@ commands.setNetworkConnection = async function (type) {
   });
   if (!airplaneMode) {
     await this.wrapBootstrapDisconnect(async () => {
-      await this.adb.setWifiAndData({wifi, data});
+      await this.adb.setWifiState(wifi);
+      await this.adb.setDataState(data);
```

Comment:
@imurchie I think we need to revert this change you have suggested as it should be the same way as,
```
await this.adb.setAirplaneMode(airplaneMode);
```
Airplane mode and Wifi state are defined the same way in appium-adb. Toggle methods take objects whereas these are not taking object as is.